contracts/MemeFactory.sol

```diff
@@ -205,7 +205,15 @@ abstract contract MemeFactory {
             deadline: block.timestamp
         });
 
-        (positionId, liquidity, , ) = IUniswapV3(uniV3PositionManager).mint(params);
+        (positionId, liquidity, amount0, amount1) = IUniswapV3(uniV3PositionManager).mint(params);
+
+        // Schedule for ascendance leftovers from native token
+        // Note that meme token leftovers will be purged later
+        uint256 nativeLeftovers = isNativeFirst ? (nativeTokenAmount - amount0) : (nativeTokenAmount - amount1);
+        if (nativeLeftovers > 0) {
+            nativeLeftovers = _launchCampaign(nativeLeftovers);
+            scheduledForAscendance += nativeLeftovers;
+        }
```

Comment:
This is an artifact of crap logic. There should be no leftovers.

Reply:
This was revealed by tests. This would mean that some super tiny amounts are just not divisible.

Reply:
Keep seeing it by Uniswap performance, there's no other way, need to revert this back.